<ui version="4.0" >
 <class>AccountManagementClass</class>
 <widget class="QWidget" name="AccountManagementClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AccountManagement</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <item row="0" column="0" >
    <widget class="QListWidget" name="m_account_list" >
     <property name="viewMode" >
      <enum>QListView::ListMode</enum>
     </property>
     <property name="modelColumn" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="addButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Add</string>
       </property>
       <property name="icon" >
        <iconset resource="../qutim.qrc" >
         <normaloff>:/icons/crystal_project/add.png</normaloff>:/icons/crystal_project/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Remove</string>
       </property>
       <property name="icon" >
        <iconset resource="../qutim.qrc" >
         <normaloff>:/icons/crystal_project/remove.png</normaloff>:/icons/crystal_project/remove.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../qutim.qrc" />
 </resources>
 <connections/>
</ui>
